mithril-test-lab/mithril-devnet/devnet-mkfiles.sh

```diff
@@ -606,19 +620,30 @@ do
                 if [ "\$POOL_STAKE_PREVIOUS_EPOCH" != "0" ] ; then
                     break
                 else
-                    echo ">>>> Not retrievable yet"
+                    echo ">>>> Stakes are not retrievable for this pool yet..."
```

Comment:
`echo ">>>> Stakes are not retrievable for this pool yet... [attempt \$POOL_STAKE_RETRIEVAL_WAIT_ROUNDS]"`